VALUES.md

```diff
@@ -0,0 +1,66 @@
+# Ayo.js Community Values
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at
+creating a new foundation of project governance and management that brings
+**humans** and their concerns front and center.
+
+You can pronounce it however you want: be it “ey-yo”, “I-oh”, or “Awooooo” — the
+Ayo.js project strives to create an environment where you can feel at home and
+focus on your contributions to this nice shared space that we all benefit from.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations,
+package management, libuv, V8 and Chakra, builds/ops, testing and its
+infrastructure, hardware and IoT, Android, streams, community events,
+facilitation, moderation and education. These and more are all concerns that
+can be handled separately and by members who might be much better qualified
+for these individual tasks than a central decision body.
+
+Ayo.js will to go further than Node.js did in spreading responsibility across
+the project, so everyone can focus and specialize better than before.
+Contributors should be able to organize and manage their own subprojects
+in a way that is effective and efficient for their unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance
+policy should change with the required agility. Following the governance policy
+to the letter will not take precedence over staying true to our stated values or
+the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with
+members with all sorts of perspectives. You are bound to find yourself in
+situations with no clear consensus, with new concerns being brought up
+on a regular basis, and things that might seem insignificant in other
+contexts suddenly exploding.
+
+The Ayo.js project’s perspective is that when such things happen, it often takes
+a toll on the well-being of the members participating, at times preventing
+repeated contributions or precluding new contributors outright. It also
+recognizes that the difference between individual choices is less important than
+the ability to come to a appropriate technical decision.
+
+Projects change, evolve, and find themselves dealing with new concerns all the
+time. In all technical matters, Ayo.js will take into account that in the end,
+it is the people making, implementing, and maintaining that matter most, not the
+decisions themselves. It’s okay for code to be imperfect.
```

Comment:
>  It’s okay for code to be imperfect.

Uhhh. I'd prefer code that ran faster and was optimized. People shouldn't feel ousted because their pull was denied because it had garbage code in it. If this fork turns into a bloated mess, people won't use it. 🤷

Reply:
.....this isn't about ousting anyone, and I have not the remotest clue where you got that.

This is about not dragging on technical conversations to the point of diminishing returns, and framing them in as constructive a way as possible. [Perfect is the enemy of good](https://en.wikipedia.org/wiki/Perfect_is_the_enemy_of_good).